save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RouterUtils.kt

```diff
@@ -44,7 +44,7 @@ fun ChildrenBuilder.withNavigate(handler: ChildrenBuilder.(NavigateFunctionConte
  * @return a function component
  */
 @Suppress("TYPE_ALIAS")
-fun <T : Props> withRouter(handler: ChildrenBuilder.(Location, Params) -> Unit) = FC<T> {
+fun withRouter(handler: ChildrenBuilder.(Location, Params) -> Unit) = FC<Props> {
```

Comment:
```suggestion
fun withRouter(handler: ChildrenBuilder.(Location, Params) -> Unit) = FC {
```

Reply:
Will also be done in later pr, I guess

Reply:
it doesn't work in old version.
I can update it now to 
```
fun withRouter(handler: ChildrenBuilder.(Location, Params) -> Unit): FC<Props> = FC {
```
but it (FC<Props>) will be removed after upgrading

Reply:
will do after upgrading together with removing `: FC<Props> = FC {`

Reply:
updated